src/components/Note.js

```diff
@@ -39,43 +60,23 @@ export const Note = ({ note }) => {
       <small className={`${color}-bg label-text p-05`}>{label}</small>
       <div className="note-footer row-flex">
         <small className="inherit-color">{formatDate(createdOn)}</small>
-        <div className="note-actions w-10rm row-flex">
-          <FontAwesomeIcon icon={faPen} onClick={() => setModalOpen(true, note)} title="Edit" />
-          <FontAwesomeIcon
-            icon={archiveNotesIndex > -1 ? faFolderOpen : faArchive}
-            onClick={() => {
-              dispatchNote({ type: "ARCHIVE_NOTE", payload: note });
-              dispatchToast({
-                type: "SHOW_TOAST",
-                payload: { state: "success", msg: `Note ${archiveNotesIndex > -1 ? "unarchived" : "archived"}` },
-              });
-            }}
-            title={archiveNotesIndex > -1 ? "Unarchive" : "Archive"}
-          />
-          <FontAwesomeIcon
-            icon={faTrashCan}
-            onClick={() => {
-              dispatchNote({ type: "DELETE_NOTE", payload: note });
-              dispatchToast({
-                type: "SHOW_TOAST",
-                payload: { state: "success", msg: `Note deleted` },
-              });
-            }}
-            title="Delete"
-          />
-          {deletedNotesIndex > -1 && (
+        <div
+          className={`note-actions w-${
+            location.pathname === "/archivesfeed" || location.pathname === "/deletedfeed" ? "5" : "10"
```

Comment:
suggestion: this condition can be refactored to improve readability, instead of writing it inline can be moved up and result can be flag to check what unit